examples/applicationcomposer-jaxws-cdi/README_pt.adoc

```diff
@@ -0,0 +1,231 @@
+= Application Composer com JAX-WS and CDI
+:index-group: Testing Techniques
+:jbake-type: page
+:jbake-status: published
+
+
+Application Composer é uma API para criar uma aplicação EE programaticamente. Ísto é muito útil em um contexto de texto, por isso não é de admirar que se originou a partir daqui.
+Para mais informação sobre o Application Composer, por favor acessar o link http://tomee.apache.org/tomee-8.0/docs/application-composer/index.html[Seção do Compositor de Aplicativos na documentação].
+
+Neste exemplo, vamos usar o `@ApplicationComposer` no contexto de uma aplicação JAX-WS .
+
+== Exemplo
+
+Este exemplo é uma aplicação de planejamemento de reuniões. É um aplicativo JAX-WS e aceita requisições para a reserva de reuniões. A aplicação primeiro verifica se a requisão pode ser reservada, se possivel, ela é reservada e retorna para o cliente se o pedido foi reservado (um valor booleano). Esta é uma aplicação simples, com propósito de demonstração, por isso é bastante simplificado. Uma solicitação de reserva é representada apenas pela data de início da reunião (um valor `java.util.Date`).
```

Comment:
Hi Daniel, in the sentence "é uma aplicação de planejamemento" the correct word is "planejamento".

Reply:
done